src/browser/wpe/opencdm/open_cdm.h

```diff
@@ -73,7 +73,7 @@ int Update(uint8_t*, int, std::string&);
   void SelectKeySystem(const std::string& );
   void SelectSession(const std::string& );
   bool IsTypeSupported(const  std::string& keySystem,const  std::string& mimeType);
-  int Decrypt(unsigned char*, uint32_t, unsigned char*, uint32_t);
+  int Decrypt(unsigned char*, uint32_t, unsigned char*, uint32_t, uint32_t *, uint32_t, int, uint32_t);
```

Comment:
@ajutras-linaro Are we changing WPE gstreamer decryptor code as well?

Reply:
Yes, you can see the proposed changes at: https://github.com/ajutras-linaro/WebKitForWayland/pull/1. Thinking about it, there are better options to handle the change in API (example: function overloading). I will implement a better solution for the next patch set.